pypicloud/cache/dynamo.py

```diff
@@ -22,6 +26,86 @@
 LOG = logging.getLogger(__name__)
 
 
+def _decide_between_versions(contender, current, allow_overwrite):
```

Comment:
I don't think you need to respect `allow_overwrite` for this process. That setting is just there to prevent you from overwriting an existing package file, and I don't think it should matter at all if you're rebuilding the cache data.